### PR TITLE
 Added description and disable-extensions

### PR DESCRIPTION
**Pull request by Fihtangolz**

### CODE REVIEW

**Comment by matklad** [comment]
makes sense to do!

bors r+

File: .vscode/launch.json [hunk 1]
```diff
@@ -2,14 +2,21 @@
   // Use IntelliSense to learn about possible attributes.
   // Hover to view descriptions of existing attributes.
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+
+  // NOTE: --disable-extensions
+  // Disable all installed extensions for increas performance of playground 
+  // and prevent potential problem and collision with developer's extansion
```
Comment (lnicola) [inline .vscode/launch.json]:
```suggestion
   // Disable all installed extensions to increase performance of the debug instance 
   // and prevent potential conflicts with other installed extensions.
```

File: .vscode/launch.json [hunk 2]
```diff
@@ -2,14 +2,21 @@
...
+
   "version": "0.2.0",
   "configurations": [
     {
+      // Used for testing extansion that located in extensionDevelopmentPath
```
Comment (lnicola) [inline .vscode/launch.json]:
```suggestion
      // Used for testing the extension with the installed LSP server.
```

File: .vscode/launch.json [hunk 3]
```diff
@@ -18,11 +25,13 @@
       "preLaunchTask": "Build Extension"
     },
     {
+      // Used for testing extansion with local lsp server located in __RA_LSP_SERVER_DEBUG
```
Comment (lnicola) [inline .vscode/launch.json]:
```suggestion
      // Used for testing theextension with a local build of the LSP server (in `target/debug`).
```

File: .vscode/launch.json [hunk 4]
```diff
@@ -34,7 +43,13 @@
...
     },
     {
-      "name": "Debug Lsp Server",
+      // Attaching LLDB by Pid to lsp server
+      // NOTE: it requires root permission, also for better debugging
+      // if you want more than just assembler representation for ex: 
+      // walk in file, you need debugging symbols.
+      // Project can build with it, for set - chane inside
+      // ./Cargo.toml field debug 
+      "name": "Attach To Server",
```
Comment (lnicola) [inline .vscode/launch.json]:
```suggestion
      // Used to attach LLDB to a running LSP server.
      // NOTE: Might require root permissions (see `kernel.yama.ptrace_scope`).
      // Don't forget to enable debug info in `Cargo.toml` beforehand. 
```

Comment (lnicola) [reply .vscode/launch.json]: This didn't come out right, sorry.

File: .vscode/launch.json [hunk 5]
```diff
@@ -18,11 +26,13 @@
       "preLaunchTask": "Build Extension"
     },
     {
+      // Used for testing theextension with a local build of the LSP server (in `target/debug`).
```
Comment (lnicola) [inline .vscode/launch.json]:
```suggestion
      // Used for testing the extension with a local build of the LSP server (in `target/debug`).
```

Oof, I messed this up.

**Comment by Veetaha** [comment]
@matklad , I always had these setting at my local setup, maybe it makes sence to add `skipFiles` node iternals?

File: .vscode/launch.json [hunk 6]
```diff
@@ -2,14 +2,22 @@
   // Use IntelliSense to learn about possible attributes.
   // Hover to view descriptions of existing attributes.
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+
+  // NOTE: --disable-extensions
+  // Disable all installed extensions for increas performance of playground
```
Comment (Veetaha) [inline .vscode/launch.json]:
```suggestion
```

File: .vscode/launch.json (hunk 4 above)
Comment (Fihtangolz) [reply .vscode/launch.json]: This has a more precise name. When you debug, do you usually expect a launch with debugger

**Comment by Fihtangolz** [comment]
> @matklad , I always had these setting at my local setup, maybe it makes sence to add [`skipFiles` node internals](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_skipping-uninteresting-code-node-chrome)?
> image
makes sense, also we need fix duplicate https://github.com/rust-analyzer/rust-analyzer/blob/3a1c7808a423b72a2603a420518dc5daf5c7a798/.vscode/launch.json#L8 . 
Sorry, I don’t quite understand how dialogue resolution works. Maybe I did something wrong.

**Comment by lnicola** [comment]
>  Sorry, I don’t quite understand how dialogue resolution works. Maybe I did something wrong.

Yeah, no worries. We'll clean it up at some point.

**Comment by Veetaha** [comment]
@Fihtangolz see #3175

File: .vscode/launch.json (hunk 1 above)
Comment (Fihtangolz) [reply .vscode/launch.json]: In this repo we have duplicate of comment

**Pull request merged**